### ISSUE TITLE
Can't get DynamicQuantizedLSTM weight and bias

### ISSUE THREAD

**Issue by sunkr1995** [pytorch/pytorch#31468]
## 🐛 Bug

when I run https://pytorch.org/tutorials/advanced/dynamic_quantization_tutorial.html

can't get DynamicQuantizedLSTM weight , bias , scale, zero_point 

```
>>print(quantized_model.rnn.state_dict())
OrderedDict()
>>print(quantized_model.state_dict())
OrderedDict([('encoder.weight', tensor([[-0.2349,  0.4934, -0.3151,  ...,  0.4456,  0.4912,  0.1553],
        [-0.0710,  0.5101, -0.2940,  ...,  0.1747,  0.5764,  0.3247],
        [-0.0229,  0.0302,  0.0874,  ...,  0.0390,  0.0736, -0.0558],
        ...,
        [-0.0049, -0.0769, -0.0859,  ..., -0.0870,  0.0463,  0.0657],
        [ 0.0656, -0.0467,  0.0178,  ...,  0.0973,  0.0566, -0.0561],
        [ 0.0098,  0.0591, -0.0863,  ..., -0.0715, -0.0329,  0.0221]])), ('decoder.weight', tensor([[-0.2003,  0.2629,  0.3254,  ...,  0.4631, -0.3630, -0.0125],
        [-0.1502,  0.0501,  0.1627,  ...,  0.5132, -0.0626, -0.3380],
        [ 0.0000,  0.0125,  0.0626,  ..., -0.1001,  0.0626, -0.0501],
        ...,
        [-0.0751,  0.0501,  0.0626,  ...,  0.0000,  0.0751, -0.1252],
        [ 0.0501, -0.0876, -0.0501,  ...,  0.0751, -0.0626, -0.0751],
        [ 0.0376,  0.0876,  0.0626,  ..., -0.0250,  0.0876, -0.0626]],
       size=(33278, 256), dtype=torch.qint8,
       quantization_scheme=torch.per_tensor_affine, scale=0.012516804970800877,
       zero_point=0)), ('decoder.scale', tensor(1.)), ('decoder.zero_point', tensor(0)), ('decoder.bias', tensor([ 5.6618,  4.8450,  0.1365,  ..., -0.3657, -0.3338, -0.3476],
       requires_grad=True))])
```

 - PyTorch Version : 1.3.1
 - OS (e.g., Linux): Ubuntu
 - How you installed PyTorch : conda
 - Build command you used (if compiling from source): conda install pytorch torchvision cudatoolkit=10.1 -c pytorch
 - Python version: 3.7.5

cc @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a

**Comment by z-a-f** [comment]
Looking into it. Meanwhile, you can get the parameters by running 

```
for param in quantized_model.rnn._all_weight_values:
  print(torch.ops.quantized.linear_unpack(param.param))
```

**Comment by z-a-f** [comment]
After #31540 lands you can see the weight values after running `state_dict()` on it. You can also access all the weights using `quantized_model.rnn.all_weights`

**Comment by ssykiotis** [comment]
this issue is fixed only in Pytorch 1.5, from 1.6 onwards the issue persists and the state_dict() of a quantized LSTM layer retuns C Scriptobjects instead of weights
